guides/source/action_mailer_basics.md

```diff
@@ -634,9 +634,13 @@ As the `:asset_host` usually is consistent across the application you can
 configure it globally in `config/application.rb`:
 
 ```ruby
-config.asset_host = 'http://example.com'
+config.action_mailer.asset_host = 'http://example.com'
 ```
 
+NOTE: Because we can't infer the protocol from the request, you'll need to
+specify a protocol or "scheme" such as `http://` or `https://.` in the
```

Comment:
```suggestion
specify a protocol such as `http://` or `https://.` in the
```

Nit -- let's be concise and just call it a protocol?

Reply:
I'm good with that. I was just mirroring other parts of the documentation.